Makefile

```diff
@@ -22,14 +22,18 @@ ESCAPE_SLASH=/
 
 # assuming we're running under msys2 (git-bash), PATH needs /c/foo format directories (because
 # it itself is :-delimited)
-PATH_PREFIX=$(shell cygpath.exe -u $(PREFIX))
+DESTDIR=$(abspath build/install)
+BUILD_PREFIX=$(DESTDIR)/wasi-sdk
 
-endif
+else
 
-PREFIX?=/opt/wasi-sdk
-PATH_PREFIX?=$(PREFIX)
+PREFIX=/opt/wasi-sdk
```

Comment:
Same here, I configure this prefix in a downstream package for an internal deb build, so I'd prefer to keep it with `?=`